### PR TITLE
Feature/core 1032 container lists feedback

### PR DESCRIPTION
**Pull request by karladler**

### CODE REVIEW

File: core/evancore.vue.libs/src/i18n/de.json [hunk 1]
```diff
@@ -110,6 +110,7 @@
     },
     "need-an-account": "Noch kein Account? <a href=\"#/dashboard.vue.evan/onboarding.vue.evan/sign-up\">Account erstellen</a>",
     "no": "Nein",
+    "no-permission": "Keine Berechtigung zur Anzeige",
```
Comment (Tschuck) [inline core/evancore.vue.libs/src/i18n/de.json]: Sounds a bit weirdo. Maybe something like this?

```suggestion
    "no-permission": "Sie sind nicht berechtigt den Inhalt einzusehen.",
```

File: dapps/digital-twin-detail/src/components/sharing/SharingOverview.vue [hunk 2]
```diff
@@ -94,7 +94,15 @@
...
 
           <template v-slot:cell(containers)="data">
-            {{ data.item.containerNames.join(', ') }}
+            <!--TODO:  :title="data.item[containerName].fields.join(', ')" -->
+            <span
+              v-for="(containerName, idx) in data.item.containerNames"
+              :key="containerName"
+              :title="data.item.containerFields[data.item.containers[idx]]"
+            >
+              {{ (containerName !='' && idx != 0) ? ', ' : '' }}
```
Comment (datrinh) [inline dapps/digital-twin-detail/src/components/sharing/SharingOverview.vue]: This is rather ugly to read in a template. Refactoring it into a helper function would also help with the TODO above :)

File: dapps/digital-twin-detail/src/components/sharing/SharingOverview.ts [hunk 3]
```diff
@@ -132,21 +138,36 @@ export default class SharingOverview extends mixins(EvanComponent) {
...
-              containers: [],
-            };
-          }
-
-          permittedUsers[address].containers.push(containerAddress);
+      unique.forEach(async (permittedUserAddress: string): Promise<void> => {
+        if (!permittedUsers[permittedUserAddress]) {
+          permittedUsers[permittedUserAddress] = {
+            type: profileUtils.getProfileType(runtime, permittedUserAddress),
+            name: profileUtils.getUserAlias(runtime, permittedUserAddress),
```
[the remark this answers is not in the document]
Comment (karladler) [reply dapps/digital-twin-detail/src/components/sharing/SharingOverview.ts]: I don't know, didn't changed anything in the logic, maybe it just works, bc it doesn't hurt when it resolves a bit later ...

File: dapps/digital-twin-detail/src/components/sharing/SharingOverview.vue (hunk 2 above)
Comment (karladler) [reply dapps/digital-twin-detail/src/components/sharing/SharingOverview.vue]: this is actually the TODO above, just forgot too remove it ;) 
I can't think of any helper function to do this properly. We would rather need a new component.

Comment (datrinh) [reply dapps/digital-twin-detail/src/components/sharing/SharingOverview.vue]: OK, then at least this :p 
```suggestion
              {{ (containerName != '' && idx != 0) ? ', ' : '' }}
```

File: dapps/digital-twin-detail/src/components/sharing/SharingOverview.ts (hunk 3 above)
[the remark this answers is not in the document]
Comment (datrinh) [reply dapps/digital-twin-detail/src/components/sharing/SharingOverview.ts]: OK, I dont feel great about it, but as long as it works :)

**Pull request merged**